components/Article.js/Article.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import useSWR from 'swr';
+import { StyledH1, StyledImage, StyledButton, StyledContainer } from "./styles";
+
+const fetcher = (url) => fetch(url).then((res) => res.json());
```

Comment:
Wrap your `App` inside a `<SWRConfig>` provider and pass the fetcher there as `value` - [see docs](https://swr.vercel.app/docs/global-configuration).